tests/tests_unit/graph/test_loaders/test_dms_loader.py

```diff
@@ -65,3 +65,4 @@ def test_imf_attribute_nodes():
 
     assert len(knowledge_nodes) == 56
     assert knowledge_nodes[0].sources[0].properties["predicate"].startswith("http")
+    assert len(store.multi_type_instances) == 63
```

Comment:
These types of tests are typically hard to debug when they go wrong. Ideally, smaller example, use data regression, or test for exact output.